lib/pharos/resources/weave/daemon-set.yml.erb

```diff
@@ -82,6 +82,17 @@ spec:
           volumeMounts:
             - name: xtables-lock
               mountPath: /run/xtables.lock
+        - name: weave-flying-shuttle
+          env:
+            - name: HOSTNAME
```

Comment:
uh, sounds bad idea to use `HOSTNAME`. wouldn't that "override" the hostname seen in the container?

Reply:
See the other containers in this pod...

Reply:
right, and it's in host network anyway.